src/translations/de.ts

```diff
@@ -255,4 +255,20 @@ export const de: Translations = {
   duplicate: 'Klonen',
   connect: 'Verbinden',
   disconnect: 'Trennen',
+
+  compas: {
```

Comment:
I see a lot of placeholders, OpenSCD doesn't work without the keys?

Reply:
Correct, the compiler complains if there are not there. So some should make the German translation some time.